tests/geo_commands_test.py

```diff
@@ -83,105 +87,143 @@ def test_geopos(redis):
 def test_georadius(redis):
     res = yield from redis.geoadd(
         'geodata',
-        13.361389, 38.115556,
-        'Palermo', 15.087269, 37.502669, 'Catania'
+        13.361389, 38.115556, 'Palermo', 15.087269, 37.502669, 'Catania'
     )
     assert res == 2
 
     res = yield from redis.georadius(
         'geodata', 15, 37, 200, 'km', encoding='utf-8'
     )
     assert res == [
-        ['Palermo'], ['Catania']
```

Comment:
@polyn0m, I was thinking about it and still sceptical.
We must follow Redis `georadius` return convention, ie:
return list of strings when no `withcoord / withhash / withdist` flags are set;
return list of (named)tuples when any of above flags are set.

In cases when no `WITH*` flags are set distance, coordinate and hash fields
of tuple would be empty and the only usable field would be name.
User would need to convert such list of one-item-tuples to plain list
to use it (for instance to do join — `', '.join(['Palermo', 'Catania'])`,
or to pass this results to other DB query...)

In case we always return list of tuples there would be situations when double conversion
happen — aioredis convert plain list to list of tuples; then user converts list of tuples to plain list.
I think this would be uncommon case but it (returning plain list of strings) would
make code more readable and simplier (both library- and user- sides).